lib/phoenix/controller.ex

```diff
@@ -704,4 +704,16 @@ defmodule Phoenix.Controller do
   defp persist_flash(conn, value) do
     put_private(conn, :phoenix_flash, value)
   end
+
+  @doc """
+  Invokes the controller plug and calls its action
+
+  ## Examples
+
+      iex> Controller.call_action(conn, MyApp.MyController, :index)
+
+  """
+  def call_action(conn, controller, action) do
```

Comment:
I don't like this because it hides they are just plugs. :( Maybe a convenience we could add in the future but I think it is an important message to get across.